cqf-fhir-benchmark/src/test/java/org/opencds/cqf/fhir/benchmark/plandefinition/TestPlanDefinition.java

```diff
@@ -433,11 +433,10 @@ public GeneratedBundle hasQuestionnaireResponse() {
         @SuppressWarnings({"unchecked", "squid:S2259"})
         public GeneratedBundle hasQuestionnaireResponseItemValue(String linkId, String value) {
             var answerPath = modelResolver.resolvePath(items.get(linkId), "answer");
-            var answers = answerPath instanceof List<?>
-                    ? ((List<?>) answerPath)
-                            .stream()
-                                    .map(a -> (IPrimitiveType<String>) modelResolver.resolvePath(a, "value"))
-                                    .toList()
+            var answers = answerPath instanceof List<?> l
+                    ? l.stream()
```

Comment:
Question:  How did you find these?  Was it sonar, vscode warnings, eyeballing the code, or some other tool?